pyproject.toml

```
@@ -0,0 +1,15 @@
+[project]
+name = "comfyui_isaacnodes"
+description = "This extension offers automatic drums extraction from audio files, as well as a few helper nodes to support my audio synchronization AnimateDiff workflows."
+version = "1.0.0"
+license = "LICENSE"
```

```suggestion
license = {file ="COPYING"}
```

Like https://github.com/Comfy-Org/comfy-cli/blob/9ea503d175e8f74a7931f824598ab4d929043e59/pyproject.toml#L7?

---

The value of the license field should just be the file path to your license

---

So if you have COPYING as your license file name in your root directory then, the license value should just be “COPYING” or “LICENSE” , etc